playbooks/tasks/lastpass-load-json-note.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Load yaml secure note from LastPass to Ansible variable
+
+# This tasks is supposed to be called (included) in a loop with list of
+# dictionaries defining lastpass_path and var_name.
+# Example:
+#
+# - name: Load LastPass data
+#   include_tasks: tasks/lastpass-load-yaml-note.yml
+#   loop:
+#     - lastpass_path: "Shared-Data/abc.yml"
+#       var_name: abc
+#     - lastpass_path: "Shared-Data/other-var.yml"
+#       var_name: some_other_var_name
+
+# Output:
+# This task sets output variable (var_name) value according to the data from
+# the given LastPass path (lastpass_path).
+#
+# - If the given path doesn't exists in LastPass,
+#   the output variable is not defined.
+# - If the secured note at the given path in LastPass is empty
+#   the ouptput variable is empty dictionary, i.e. {}.
+# - Otherwise yaml data from secure note are loaded into output variable.
+
+- name: Check if LastPass item exists
+  local_action:
+    module: ansible.builtin.command
+    cmd: "lpass ls {{ item.lastpass_path }}"
+  register: lastpass_ls_result
+  changed_when: False
+
+- block:
+    - name: Get LastPass data
+      set_fact:
+        lastpass_data: "{{ lookup('community.general.lastpass', item.lastpass_path, field='notes') | from_json }}"
```

Comment:
@firyx the only reason I duplicated this file from `lastpass-load-json-note.yml` is because I couldn't get the `from_json` and `from_yaml` to be handled properly when they were passed in as `item.format` for example. 

Not a big deal, but a potential follow up to DRY up these two files.